lib/matplotlib/animation.py

```diff
@@ -1057,6 +1060,18 @@ def func(current_frame: int, total_frames: int) -> Any
             _log.info("Disabling savefig.bbox = 'tight', as it may cause "
                       "frame size to vary, which is inappropriate for "
                       "animation.")
+
+        facecolor = savefig_kwargs.get('facecolor',
+                                       mpl.rcParams['savefig.facecolor'])
+        if facecolor == 'auto':
+            facecolor = self._fig.get_facecolor()
```

Comment:
We should be able to test this code path in the same `NullMovieWriter` test below, no?

Reply:
Done, but this is still just a smoke test 😞